dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -1259,7 +1259,7 @@ Spdp::remove_expired_participants()
     DiscoveredParticipantIter part = participants_.find(*participant_id);
     if (part != participants_.end()) {
       const MonotonicTimePoint expr(MonotonicTimePoint::now() -
-        rtps_time_to_time_duration(part->second.pdata_.leaseDuration));
+        rtps_duration_to_time_duration(part->second.pdata_.leaseDuration, part->second.pdata_.participantProxy.protocolVersion, part->second.pdata_.participantProxy.vendorId));
```

Comment:
Please avoid lines that are this long.  It is in the guidelines, and our past experience is that someone else comes along later and changes it.

Reply:
Can we just bite the bullet and pick & use a style linter as part of the PR process already?

Reply:
> Please avoid lines that are this long. It is in the guidelines, and our past experience is that someone else comes along later and changes it.

I feel accused :smile: 

> Can we just bite the bullet and pick & use a style linter as part of the PR process already?

I have a line length check as part of #1366. Right now it's set at 150, but it's marked as a TODO to determine a line length. I would like the limit to be 120, but that matches a lot of files. As a interesting side note, it actually had a false positive on a line Tim wrote in Greek and Hebrew for the Rapidjson TS test. I changed it to convert to UTF-32, then check the length.

An actual proper style linter would be nice though.